UnityProject/Assets/Scripts/Core/Editor/DisableNonEssentialScenes.cs

```diff
@@ -0,0 +1,215 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using System.Linq;
+using System.IO;
+
+/// <summary>
+/// This editor tool allows to disable some scenes from build
+/// It will drasticly improve build time
+/// </summary>
+public class DisableNonEssentialScenes : EditorWindow
+{
+	private static List<string> AdditionalScenesToRemove = new List<string>()
+	{
+		"Fallstation Centcom",
+		"Fallstation Syndicate"
```

Comment:
I think with `NaughtyAttributes` you can serialize refererences to scenes. Would that be a better solution?
It seems "WizardShip and "WizardAsteroid" additional scenes remain enabled. Is this correct?

Reply:
Yeah, in ideal situation, one of them should be disabled. I just haven't done yet, because they need some tweaks in additional scenes scriptable object. Same with Fallstation Centcom-Syndicate.
It's relatively easy to add text field or NaughtyAttributes for additional scene to disable. Just haven't done it yet.